examples/hyperparameter/run_benchmark.py

```diff
@@ -0,0 +1,51 @@
+"""
```

Comment:
probably should get rid of this comment. if it's important, then put it in the tutorial.